control-plane-operator/controllers/hostedcontrolplane/config/constants.go

```diff
@@ -1,7 +1,8 @@
 package config
 
 const (
-	DefaultPriorityClass         = "system-node-critical"
+	APICriticalPriorityClass     = "hypershift-api-critical"
```

Comment:
nit: can we add a comment with a short definition of these classes clarifying the different nuances between both? why a control plane component should use one over the other one?